bolt/src/main/java/com/slack/api/bolt/AppConfig.java

```diff
@@ -141,30 +156,121 @@ public void setOauthCallbackPath(String oauthCallbackPath) {
 
     private String appPath;
 
+    // --------------------------
+
+    @Deprecated // will be removed in v2.0 - use getOauthInstallRequestURI instead
     public String getOauthStartRequestURI() {
+        return getOauthInstallRequestURI();
+    }
+
+    public String getOauthInstallRequestURI() {
         if (appPath == null) {
-            return oauthStartPath;
+            return getOauthInstallPath();
         } else {
-            return appPath + oauthStartPath;
+            return appPath + getOauthInstallPath();
         }
     }
 
+    // --------------------------
+
+    @Deprecated // will be removed in v2.0 - use getOauthRedirectUriRequestURI instead
     public String getOauthCallbackRequestURI() {
+        return getOauthRedirectUriRequestURI();
+    }
+
+    public String getOauthRedirectUriRequestURI() {
```

Comment:
I know this name is a bit weird but, as `RequestURI` is a common term to describe the path of HTTP requests in Java, I kept choosing this suffix here.